modules/calib3d/src/calibinit.cpp

```diff
@@ -1631,7 +1631,7 @@ void ChessBoardDetector::findQuadNeighbors()
                         // check edge lengths, make sure they're compatible
                         // edges that are different by more than 1:4 are rejected
                         float ediff = cur_quad.edge_len - q_k.edge_len;
-                        if (ediff > 32*cur_quad.edge_len ||
+                        if (-ediff > 32*cur_quad.edge_len ||
```

Comment:
If `q_k.edge_len` > `cur_quad.edge_len` then `ediff` < 0 and condition `ediff > 32*cur_quad.edge_len` is always `false`

Reply:
why not `ediff = abs(..)`?

Reply:
Added minus instead of calling `abs` function for better performance. But now profile it and time difference is not noticeable at all